src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -185,6 +185,14 @@ private boolean lineMatching(int currentLine) {
         return result;
     }
 
+    /**
+     * Check whether one line in lineRangeList is between
+     * event ast node's child nodes' min and max lineNo.
+     *
+     * @param childAstStartLine event ast node's child nodes' min lineNo
+     * @param childAstEndLine event ast node's child nodes' max lineNo
+     * @return true if one line is between min and max lineNo.
```

Comment:
`min and max` - please refer to parameters childAstStartLine and childAstEndLine